app/bundles/LeadBundle/Resources/views/Lead/lead_stats.html.twig

```diff
@@ -16,4 +16,11 @@
     </div>
     {% endfor %}
 </div>
-{% endif %}
\ No newline at end of file
+{% if emailStats['sent_count'] == 0 and
+    emailStats['open_rate'] == 0 and
+    emailStats['click_through_rate'] == 0 and
+    emailStats['click_through_open_rate'] == 0
+%}
+    {{ include('@MauticCore/Helper/no_information.html.twig', {'tip': 'mautic.contact.noinformation.statistics'|trans}) }}
+{% endif %}
+{% endif %}
```

Comment:
I can see 1 `if` being opened but 2 `endif`. What's up with that?

Reply:
The first ends the if started at line 19
The second ends the started at line 1

Reply:
Gotcha. The indentation helps.